bedrock/firefox/views.py

```diff
@@ -704,6 +704,7 @@ class NewView(L10nTemplateView):
         'firefox/new/trailhead/download-yandex.html': ['firefox/new/download', 'banners/firefox-mobile'],
         'firefox/new/desktop/download.html': ['firefox/new/desktop'],
     }
+    ftl_activations = ['firefox/new/download', 'firefox/new/desktop']
```

Comment:
Should we also add this to [DownloadThanksView](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/views.py#L660), since that URL shares the same localizations file(s) as /new?

Reply:
Good catch. Done.